<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>userWindowClass</class>
 <widget class="QMainWindow" name="userWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>userWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="option_list_user">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>What do you want to do today?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="showButtonUser">
            <property name="text">
             <string>See pets one by one</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="specificBreedButton">
            <property name="text">
             <string>See a specific breed with a maximum given age</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="adoptionButton">
          <property name="text">
           <string>See current adoption list</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>The file name you want to save the data in (optional):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dataSave"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="closeUserWindow">
            <property name="text">
             <string>Close window</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="show_all">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Pets' name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="nameShow">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Pets' breed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="breedShow">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Pets' age:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ageShow">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Pets' photo link:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="photoShow">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QPushButton" name="previousButton">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="adoptButton">
          <property name="text">
           <string>Adopt current pet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButtonUser_2">
          <property name="text">
           <string>Back to main page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="see_by_breed">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Pets' name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="nameShow_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Pets' age:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="ageShow_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Pets' photo link:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="photoShow_2">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Searched breed:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchedBreed"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Maximum searched age:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchedAge"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchButton">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QPushButton" name="previousButton_2">
            <property name="text">
             <string>Previous</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="nextButton_2">
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="adoptButton_2">
          <property name="text">
           <string>Adopt current pet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButtonUser_3">
          <property name="text">
           <string>Back to main page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="adoption_list">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QListWidget" name="adoptionShow"/>
        </item>
        <item>
         <widget class="QPushButton" name="removeAdoption">
          <property name="text">
           <string>Give up selected pet adoption</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="backButtonUser_4">
          <property name="text">
           <string>Back to main page</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeUserWindow</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>close_window()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>831</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>537</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showButtonUser</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>show_pet_list_page()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specificBreedButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>show_specific_breed_page()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>706</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adoptionButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>show_adoption_list_page()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>568</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousButton_2</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>previous_specific_breed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton_2</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>next_specific_breed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>627</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>search_breed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButtonUser_2</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>506</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButtonUser_3</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>backButtonUser_4</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previousButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>previous_pet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>next_pet()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>583</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeAdoption</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>remove_selected_adoption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adoptButton_2</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>adopt_current()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>732</x>
     <y>517</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>adoptButton</sender>
   <signal>clicked()</signal>
   <receiver>userWindowClass</receiver>
   <slot>adopt_current()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>730</x>
     <y>513</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>show_pet_list_page()</slot>
  <slot>show_specific_breed_page()</slot>
  <slot>show_adoption_list_page()</slot>
  <slot>next_specific_breed()</slot>
  <slot>previous_specific_breed()</slot>
  <slot>search_breed()</slot>
  <slot>back()</slot>
  <slot>previous_pet()</slot>
  <slot>next_pet()</slot>
  <slot>remove_selected_adoption()</slot>
  <slot>adopt_current()</slot>
  <slot>close_window()</slot>
 </slots>
</ui>
